### ISSUE TITLE
python.flask.security.audit.directly-returned-format-string.directly-returned-format-string FP

### ISSUE THREAD

**Issue by nbrahms** [returntocorp/sgrep-rules#1034]
**Describe the bug**
If a route returns `return ("a" + "b" + "c")`, this rule will fire:

See https://semgrep.dev/s/LrPL

**Comment by minusworld** [comment]
Turns out this is an issue with the Python parser not understanding raw string, e.g., `r"some chars"`.

See https://semgrep.dev/s/RXkK.

Opening a new issue on the Semgrep repo.

**Comment by minusworld** [comment]
Blocked on https://github.com/returntocorp/semgrep/issues/3881
